packages/bindgen/spec.yml

```diff
@@ -655,7 +655,7 @@ classes:
       get_column_type: '(column: ColKey) const -> DataType'
       get_link_target: '(column: ColKey) const -> ConstTableRef'
       get_object: '(key: ObjKey) const -> Obj'
-      try_get_object: '(key: ObjKey) const -> Obj'
+      try_get_object: '(key: ObjKey) const -> Nullable<Obj>'
```

Comment:
What this an unrelated issue?

Reply:
> Was this an unrelated issue?

Sort of, sort of not. I added `Nullable<>` on `Obj::get_linked_object` to tell it to return `null` rather than a null-valued `Obj` object. When I did that, I searched for `-> Obj` to see if there were any other places where we were returning potentially null `Obj` objects. I think this was the only one, but there were a few I wasn't 100% sure about.

> I can't recall the exact semantics of Nullable.

* When going from C++ to JS: Adds a check for `!val` and returns `null` if it is true. In precise C++ terminology, requires that the type is contextually convertible to bool and results in a SDK language `null` value if it is falsey.
* When going from JS to C++: If the js value is `null` (not `undefined`), it will produce a value-constructed (ie `T()`) object of the type.

These operations are expected to be coherent, so `T()` should be falsey. I don't think we check for this, but things would be weird otherwise.

`Obj` meets these requirements.

Reply:
Thanks a lot!